docs/resources.md

```diff
@@ -1048,6 +1056,70 @@ Data,
   }
 ```
 
```

Comment:
It would also be helpful to include a section on how to use the API in the meantime. What are the recommended ways to use Tasks and Pipelines now, if you were relying on PipelineResources before?

Maybe this is already covered in a v1alpha to beta migration guide.

Reply:
Good catch, it's covered there and I've added links to it.